source-control/pull-request-templates/azure-devops/azure-devops-arch-design-pr-template.md

```diff
@@ -0,0 +1,35 @@
+# [Work Item](./link-to-the-work-item)
```

Comment:
we are using design templates when making architecture decision, which have all the sections in this template already in theirs. That would make them redundant. If we are making architecture decisions, these should all be part of the check in document/md you are already making (https://github.com/microsoft/code-with-engineering-playbook/blob/master/design-reviews/recipes/milestone-epic-design-review-template.md).

Reply:
Good point, I will remove this template